src/content/Form/Form.tsx

```diff
@@ -1,17 +1,29 @@
-import { ISectionData } from "../App/App.types"
-import "./Form.css"
-import { useContext } from "react"
-import { ModalDispatchContext, ModalPreset } from "../ModalLayer"
+import { ISectionData } from "../App/App.types";
+import "./Form.css";
+import { useContext, useState } from "react";
+import { ModalDispatchContext, ModalPreset } from "../ModalLayer";
+import SyncSavedSchedules from "../SyncSavedSchedules/SyncSavedSchedules";
 
 interface IProps {
-  newSection: ISectionData | null
-  sectionConflict: boolean
-  handleAddNewSection: () => void
-  handleCancel: () => void
+  sections: ISectionData[];
+  newSection: ISectionData | null;
+  sectionConflict: boolean;
+  handleAddNewSection: () => void;
+  handleCancel: () => void;
 }
 
 const Form = (props: IProps) => {
-  const dispatchModal = useContext(ModalDispatchContext)
+  const dispatchModal = useContext(ModalDispatchContext);
+  const [isSyncComponentVisible, setIsSyncComponentVisible] = useState(false);
```

Comment:
this state isn't necessary anymore, the modal layer takes care of visibility once a modal is posted. same with the handlers below.